### PR TITLE
Add details about team member (Khang Tran) to About-Us

### PR DESCRIPTION
**Pull request by lennoxtr**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -29,14 +29,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Khang Tran
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lennox.png" width="200px">
```
Comment (hingen) [inline docs/AboutUs.md:34]: Perhaps change "lennox.png" to "lennoxtr.png"

File: docs/AboutUs.md [hunk 2]
```diff
@@ -29,14 +29,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/lennoxtr)]
+[[portfolio](team/lennox.md)]
```
Comment (hingen) [inline docs/AboutUs.md:37]: Perhaps rename "lennox.md" to "lennoxtr.md"

File: docs/AboutUs.md [hunk 3]
```diff
@@ -29,14 +29,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: In charge of Storage
```
Comment (hingen) [inline docs/AboutUs.md:40]: Consider changing to just "File I/O"?

**Comment by lennoxtr** [comment]
I think I updated my request?

**Review by hingen — APPROVED** [review]
LGTM

**Pull request merged**